src/views/v2/tickets/item.php

```diff
@@ -50,18 +47,16 @@
 	get_post_class( '', $ticket->ID ),
 ];
 
-$data_available      = 0 === $tickets_handler->get_ticket_max_purchase( $ticket->ID ) ? 'false' : 'true';
-$data_has_shared_cap = $has_shared_cap ? 'true' : 'false';
-$ticket_item_id      = 'tribe-';
-$ticket_item_id     .= ! empty( $is_modal ) ? 'modal' : 'block';
-$ticket_item_id     .= '-tickets-item-' . $ticket->ID;
+$ticket_item_id = 'tribe-';
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space

<sub>hash: 8a719afebb75d285383ccbd1d7e1aab4</sub>